Sources/CompilerPluginSupport/TargetExtensions.swift

```diff
@@ -16,14 +16,14 @@ public extension Target {
     @available(_PackageDescription, introduced: 999.0)
     static func macro(
         name: String,
-        group: TargetGroup,
+        group: TargetGroup?,
```

Comment:
`group: TargetGroup? = .package`

Reply:
this should work like https://github.com/apple/swift-package-manager/pull/6365/files/381f87fda3ec29028aab6891acbaeb24e3398e6e#diff-1c897bc83b443b3ce4e3559868b51b1999bbefb6812fbf97229302be902eecfbR584